scenes/game_elements/characters/enemies/guard/components/guard.gd

```diff
@@ -437,35 +431,6 @@ func edit_patrol_path() -> void:
 		editor_interface.edit_node.call_deferred(patrol_path)
 
 
-func _draw() -> void:
-	## Only draw the patrol path debug line if we are in the editor
-	## and the guard node is selected.
```

Comment:
I think this change makes sense. I looked into whether one can change the color drawn in the editor for a Path2D in a different way. The answer is "kind of". Path3D has a `debug_custom_color` but Path2D doesn't: https://github.com/godotengine/godot-proposals/issues/12108#issuecomment-2766572610. The best available workaround is using self_modulate but that doesn't really work because it modulates the default green colour.

If `Path2D.debug_custom_color` existed then we could readd a similar behaviour by adjusting that colour when the PathFollowBehavior is selected in the editor, and resetting it when deselected.